java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
-  private <T extends EppResource> void importEppResource(final T resource, final String type) {
+  public <T extends EppResource> ImmutableSet<Object> getIndexesForEppResource(T resource,
```

Comment:
Note https://github.com/google/nomulus/pull/57/files#diff-3385ce686547e7ce6c33380a1672d441R181 and https://github.com/google/nomulus/pull/57/files#diff-3385ce686547e7ce6c33380a1672d441R217